Tella/Utils/Localizable/LocalizableSettings.swift

```diff
@@ -110,6 +120,18 @@ enum LocalizableSettings: String, LocalizableDelegate {
     case UwaziSuccess = "Setting_Server_Uwazi_Connect_Server"
     case UwaziSuccessMessage = "Setting_Server_Uwazi_Success_Message"
     
+    //GDrive
+    case GDriveSelectTypeToolbar = "Setting_Server_GDrive_Select_Type_Toolbar"
+    case GDriveSelectTypeTitle = "Setting_Server_GDrive_Select_Type_Title"
+    case GDriveSelectTypeDesc = "Setting_Server_GDrive_Select_Type_Desc"
+    case GDriveSelectTypeShared = "Setting_Server_GDrive_Select_Type_Shared"
+    case GDriveSelectTypePersonal = "Setting_Server_GDrive_Select_Type_Personal"
+    case GDriveSelectTypeMoreInfo = "Setting_Server_GDrive_Select_Type_More_Info"
+    case GDriveSelectSharedDriveToolbar = "Setting_Server_GDrive_Select_Shared_Toolbar"
+    case GDriveCreatePersonalFolderTitle = "Setting_Server_GDrive_Create_Personal_Title"
+    case GDriveCreatePersonalFolderDesc = "Setting_Server_GDrive_Create_Personal_Desc"
+    case GDriveCreatePersonalFolderPlaceholder = "Setting_Server_GDrive_Create_Personal_Placeholder"
+    case GDriveSuccessMessage = "Setting_Server_GDrive_Success_Message"
     // About & Help
```

Comment:
Defining the cases should be started by a lowercase letter e.g:
`gDriveSelectTypeToolbar`